packages/components/Notification/src/Notification.tsx

```diff
@@ -49,7 +49,7 @@ export const Notification = compose<NotificationType>({
     }, ['isBar']);
 
     return (final: NotificationProps, ...children: React.ReactNode[]) => {
-      const { variant, icon, title, action, onActionPress, ...rest } = mergeProps(userProps, final);
+      const { variant, visible, icon, title, action, onActionPress, ...rest } = mergeProps(userProps, final);
```

Comment:
This doesn't seem to be used anywhere. Without the animation, does this prop actually hide the notification?

Reply:
Oops, this was remaining from when visible was in Notification.tsx. Will remove